packages/frontend/src/components/Game/GameBoard/Tile/style.scss

```diff
@@ -12,3 +12,9 @@
     width: 100%;
   }
 }
+
+.matched {
+  img {
+    display: none;
+  }
+}
```

Comment:
The `img` selector could be removed if the `matched` class was separated from the others in the Tile component.

```javascript
[
  'tile border border-primary',
  tile.isMatched ? 'matched' : null
]
```